Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -694,10 +696,67 @@ def get_notable_sequence_event_types_request(self, asset_sequence_id: str = None
                                       params=params)
         return response
 
+    def get_list_incidents(self, query, incident_type, priority, status, limit=None, page_number=None, page_size=None):
+        params = self.build_incident_response_query_params(incident_type, priority, query, status, limit, page_number,
+                                                           page_size)
+        return self._http_request('GET', url_suffix='/ir/api/incident/list',
+                                  params=params)
+
+    def build_incident_response_query_params(self, incident_type, priority, query,
```

Comment:
is this only used in `get_list_incidents`? if so, please make it an inner method.